client/src/components/CKANRegistryDisplay.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Griddle from 'griddle-react';
+import classnames from 'classnames';
+// import i18n from 'i18n'; // TODO how to resolve admin module from 'not-admin'
```

Comment:
You add the standalone i18n bundle from admin - example from userforms:

```php
// Adds the i18n library
Requirements::javascript('silverstripe/admin:client/dist/js/i18n.js');
// Adds the userforms translation files
Requirements::add_i18n_javascript('silverstripe/userforms:client/lang');
```

I guess we can add this to the minimal SilverStripe templates we'll have for this module.

From here, the `i18n` object is available in the window object globally, so in this component you can declare it as a global property and use it that way.

Reply:
Nice, thanks!